src/components/layout/MainLayout/MainLayout.styled.js

```diff
@@ -7,4 +7,5 @@ export const StyledMainLayout = styled.main`
   padding: 3rem;
   border-radius: 15px;
   height: 80vh;
+  overflow-y: scroll;
```

Comment:
Maybe is better to delate overflow-y from here.

Reply:
Thank you, I checked and I left it there by accident, the table is scrollable now without scroll on the layout element.